### ISSUE TITLE
Why Can't I join MH?

### ISSUE THREAD

**Issue by Cynthix** [Minehut/Issues#155]
When I tried to join MH a few days ago, it said that I was kicked whilst connecting to a lobby, and the reason was "You are not allowed to join." I don't have any malicious intent on MH, nor do I have any programs(I.E: Hacked Clients, Macros, etc.), that could cause harm to any other players experience. It would be highly appreciated if a moderator or administrator could respond to this and resolve this issue as quickly as possible.
FYI: My MC Username is: CosmicCourage

**Comment by tjhubz** [comment]
Are you still having this issue?

**Comment by MatrixTunnel** [comment]
@HGDesertTeam ^

**Comment by HGDesertTeam** [comment]
Yes, I am still having this issue.

**Comment by Aerh** [comment]
Are you using MCLeaks? Didn't @Trophonix recently add something to block it? ¯\_(ツ)_/¯

**Comment by HGDesertTeam** [comment]
No, I don't use alts or MCLeaks. I have never experienced using a Hacked Client before, I do have one on my PC, but I don't use it that much, and I would never use it on MH.
